src/test/kotlin/org/virtuslab/pulumikotlin/codegen/step3codegen/CodeGeneratorGenerateFunctionsTest.kt

```diff
@@ -97,7 +97,8 @@ internal class CodeGeneratorGenerateFunctionsTest {
                     sources = sourceFiles
 
                     classpaths = listOf(
-                        artifact("com.pulumi:pulumi:0.6.0"),
+                        artifact("org.virtuslab:pulumi-kotlin:0.9.4.0"),
+                        artifact("com.pulumi:pulumi:0.9.4"),
```

Comment:
Is including `com.pulumi:pulumi:0.9.4` needed now? Wouldn't `org.virtuslab:pulumi-kotlin:0.9.4.0` be enough?

Reply:
These tests don't seem to have access to the `pom.xml` of each library and they don't locate their dependencies, so they have to all be listed explicitly